### ISSUE TITLE
[Docs] Incorrect link in docs/user_guide/ide-integration.rst

### ISSUE THREAD

**Issue by HarryMWinters** [PyCQA/pylint#6533]
### Bug description

While exploring the pylint documentation I noticed a link that claims to be for VSCode but actually point to Visual Studio docs. 

## Link Location
doc/user_guide/ide-integration.rst:L18

```
- `Visual Studio Code`_, see https://docs.microsoft.com/en-us/visualstudio/python/linting-python-code,
```

## Problem

This is a link to Visual Studio docs, not Visual Studio Code.

## Solution

Update with `https://code.visualstudio.com/docs/python/linting`. I'd love a chance to submit a PR against this.

### Command used

```shell
curl https://pylint.pycqa.org/en/latest/user_guide/ide-integration.html

# JK :) I just visted the website and clicked around.
```

### Pylint output

```shell
Doc bug. Not applicable. See https://pylint.pycqa.org/en/latest/user_guide/ide-integration.html.
```

### Expected behavior

The link for visual studio code docs should take me to visual studio code docs.

### Pylint version

```shell
pylint 1.14.0-dev0
```
